snippets/snippets.json

```diff
@@ -155,5 +155,58 @@
         "const MOCK_ROUTER_PROVIDER = { provide: Router, useClass: MockRouter };",
         ""
       ]
+    },
+    "Using HttpTestingController for success request": {
+      "prefix": "jat-success-http-testing-controller-spec-expecting-one-request",
+      "description": "Jasmine Test Spec (it block) to inject controller to mock and flush requests, expecting a request matching an address and flushing with mock data without any error",
+      "body": [
+        "it('$1', () => {",
```

Comment:
Please update all it() block to be like it('should ')

Reply:
have updated it to be like it('should $1') - so that there will be tab stops once snippets are added. Let me know if that should be fine, else revert without tab stops